Tests/Tests/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>no.hyper.$(PRODUCT_NAME:rfc1034identifier)</string>
+	<string>com.pod.$(PRODUCT_NAME:rfc1034identifier)</string>
```

Comment:
@sindrenm Could be, but since you're not shipping this to the App Store, the complexity of requiring a new identifier and parsing the XML might not be needed.